problems/28.Palindrome Number.gs

```diff
@@ -0,0 +1,31 @@
+function isPalindrome(x: number): boolean {
+    if (x < 0) {
+        return false
+    }
+
+    let niz = uNiz(x)
+    let i = 0
+    let j = niz.length - 1
+
+    while (i != j && j > i) {
+        if (niz[i] != niz[j]) {
+            return false
+        }
+        i++
+        j--
+    }
+    return true
+};
+function uNiz(x: number): number[] {
+    let str: string = x.toString()
```

Comment:
This is extremely inefficient way to extract digits from string...
Try be doing this:
x%10 => will give you last digit
x = x / 10 => will remove last digit
Do the above two operation as long as X  != 0; Much faster then converting to string...

Reply:
and less memory a lot less memory